common/app/com/commercetools/sunrise/common/injection/SunriseModule.java

```diff
@@ -49,6 +49,12 @@ protected void configure() {
         return Http.Context.current();
     }
 
+    @Provides
+    @RequestScoped
+    public Http.Session httpSession(final Http.Context httpContext) {
+        return httpContext.session();
+    }
```

Comment:
Makes testing easier, since classes that need session don't need to inject the whole `Http.Context`  for each test.